SpringParking-parkingBranch/src/main/java/pl/springDemo/demo/userPayment/AmountCounter.java

```diff
@@ -0,0 +1,39 @@
+package pl.springDemo.demo.userPayment;
+
+import pl.springDemo.demo.user.UserType;
+
+public class AmountCounter {
+
+    public double countHoursAmount(UserType userType, long hours) {
+        switch (userType) {
+            case VIP_DRIVER:
+                if (hours <= 1) {
+                    return 0;
+                } else if (hours == 2) {
+                    return 2;
+                } else {
+                    for (int i = 0; i < hours; i++) {
```

Comment:
Ten kod nie działa (chodzi mi o fragment z pętlą for). Robisz return wewnątrz pętli for.